porting/finalize/UBports-official-support/port-support-requirements.rst

```diff
@@ -0,0 +1,102 @@
+==================================
+
+This document enumerates the requirements that must be met in order for devices to be deemed ship ready for UBports releases.
+To be considered ready, port maintainers MUST meet the requirements presented in this document, including any documents incorporated via reference.
+
+The use of “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” is per the IETF standard defined in :RFC:`2119`.
+
+-----
+
+Attaining exceptions
+********************
+
+Process
+-------
+
+* Device or software deviations from our core product (as described below) MUST be approved by the devrel team.
+* Device or software exceptions SHOULD be made via change request to this repository.
+* All device or software exceptions that are granted MUST be documented below at the `Ubuntu Touch device page`_ for all affected devices.
+
+Hardware support
+****************
+
+Audio
+-----
+
+* All devices MUST support audio playback for media content.
+* Phones MUST support in-call audio.
+* Phones MUST support speaker audio.
+* Tablet devices capable of in-call audio/speaker audio MUST support in-call/speaker audio.
+
+RIL
+---
+
+* All devices with RIL supported in their stock OS MUST support RIL for phone calls & data.
+* All devices with RIL supported in their stock OS SHOULD support emergency calling with a SIM inserted (112/911).
+* All devices with RIL supported in their stock OS SHOULD support emergency calling without a SIM inserted (112/911).
+* Data-only devices (defined as devices that have a RIL but do not support telephony stack due to hardware/firmware restrictions) are EXEMPTED from phone & emergency dialing requirements.
+
+Software support
+****************
+
+CVE
+---
+
+* Devices MUST support CVE patches for “high profile” exploits and vulnerabilities.
+* Devices SHOULD receive regular CVE patches to the device kernel and dependencies.
+
+Installer
+---------
+* Devices MUST support installation via `UBports Installer`_.
+
+Recovery
+--------
+
+* Devices MUST support resetting to factory defaults.
+* Devices MUST support updating via OTA.
+
+Quality of Life
+***************
+
+Commit authorship
+-----------------
+
+* All non-original commits MUST have proper authorship attribution from the source it was taken from or adapted from.
+
+Documentation
+-------------
+
+* The device MUST be listed at the `Ubuntu Touch device page`_; the device page needs to be kept up-to-date.
+
+Stability
+---------
+
+* Issues like the "screen of death" MUST NOT affect the device.
+* The device MUST NOT have abnormal battery drain.
+
+Workflow
+--------
+
+* Force pushing branches SHOULD be avoided.
+* In the event of a force pushed branch, backup branches of the pre-forced ``HEAD`` MUST be made.
+
+Exception
+*********
+
+All currently granted exceptions should be listed in the following table. To request an exception, please submit a change to this repository.
```

Comment:
I don't really understand where this list is supposed to go. Where is "this" repository?